### PR TITLE
Feature/parse api configs

### PR DESCRIPTION
**Pull request by frenchbread**
@apinf/developers Review this PR please. Api backend schema updated with required fields. Yaml -> Json conversion and populating apiBackend form with data.

### CODE REVIEW

File: both/collections/backend.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ ApiBackendsSchema = new SimpleSchema({
   },
   name: {
     type: String,
+    optional: false
```
Comment (brylie) [inline both/collections/backend.js]: Name is a required field.

File: both/collections/backendConfigurations.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ApiBackendConfigurations = new FS.Collection("apiBackendConfigurations", {
   // TODO: Determine how to upload files within the project directory without hardcoding FS path
-  stores: [new FS.Store.FileSystem("apiBackendConfigurations", {path: "~/apinf/apiConfigurations"})]
+  stores: [new FS.Store.FileSystem("apiBackendConfigurations", {path: "~/Dev/meteorApps/api-umbrella-dashboard/uploads/apiBackendConfigs"})]
```
Comment (brylie) [inline both/collections/backendConfigurations.js]: What would be a good location for the config files on a production server?

File: client/views/api_backends/api_backends.html [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <template name="apiBackends">
   <div class="api-backends-content">
     <h1>Add API</h1>
+    {{> importApiConfiguration }}
```
Comment (brylie) [inline client/views/api_backends/api_backends.html]: Lets go ahead and remove the import api configuration button from the Api Backend Config form.

File: both/collections/backendConfigurations.js (hunk 2 above)
Comment (frenchbread) [inline both/collections/backendConfigurations.js]: `project_root/uploads/apiConfigs/` i the good location i think.

**Comment by bajiat** [comment]
We have created new tasks () for exporting and importing individual API backends instead of all API Umbrella backends. This code should be partially reused for the new tasks.

**Comment by frenchbread** [comment]
Cleaned up the code and made it ready to be reused in upcomming tasks.

**Comment by frenchbread** [comment]
I created a new branch - [api-config-import](https://github.com/apinf/api-umbrella-dashboard/tree/feature/api-config-import) that handles all needed code needed to make api config file imports. Since the code is current branch was too messed up after pulling changes from develop.

Closing this PR and removing old branch. New related PR - #344

**Pull request closed without merging**